src/khoj/routers/helpers.py

```diff
@@ -185,6 +189,42 @@ async def aget_relevant_information_sources(query: str, conversation_history: di
         return [ConversationCommand.Default]
 
 
+async def aget_relevant_modes(query: str, conversation_history: dict):
+    """
+    Given a query, determine which of the available tools the agent should use in order to answer appropriately.
+    """
+
+    mode_options = dict()
+
+    for mode, description in mode_descriptions_for_llm.items():
```

Comment:
Can we just use `mode_descriptions_for_llms` directly instead of creating the new `mode_options` dictionary? Given `mode_descriptions_for_llms` is already a dictionary

Reply:
Not quite. This formats it differently so that it uses the value of the `ConversationCommand` enum is used in the input prompt.